packages/interwovenkit-react/src/pages/deposit/Fields.module.css

```
@@ -139,6 +139,17 @@
   font-weight: 600;
 }
 
+.balanceLabel {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+
+  color: var(--gray-3);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)